CONTRIBUTING.md

```
@@ -12,17 +22,17 @@ To contribute to EventStoreDB:
 2. Create a feature branch from the `master` (or release) branch.
 3. It's recommended using rebase strategy for feature branches (see more in [Git documentation](https://git-scm.com/book/en/v2/Git-Branching-Rebasing)). Having that, we highly recommend using clear commit messages. Commits should also represent the unit of change.
 4. Before sending PR to ensure that you rebased the latest source branch from the main repository.
-5. When you're ready to create the [Pull Request on GitHub](https://github.com/EventStore/EventStore/compare).
+5. When you're ready to create the [Pull Request on GitHub](https://github.com/TrogonStack/TrogonEventStore/compare).
 
 ## Documentation
 
-Documentation is located in the [`docs`](/docs) folder. It's orchestrated in the separate [documentation repository](https://github.com/EventStore/documentation). It's available online at https://developers.eventstore.com/.
+Documentation is located in the [`docs`](/docs) folder. It's orchestrated in the separate [documentation repository](https://github.com/EventStore/documentation). It's available online at https://developers.trogondb.com/.
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)